docs: mention filtered by ACLs in coordinate API (#12489)
hashicorp/consul: website/content/api-docs/index.mdx

In order to limit information leakage, this header is only present for requests
authenticated by a valid ACL token.

> Below is an example using `curl` to view the
> `X-Consul-Results-Filtered-By-ACLs` response header.

Reviewer: ```suggestion
The following example uses `curl` to view the
`X-Consul-Results-Filtered-By-ACLs` response header.
```
It's not in the style guide yet, but we want to provide guidance on directional language so that we are consistently describing the order of things. Basically, we want to standardize on "The following . . .", "In the following . . ." as opposed to using "below", "above", "next", "to the right", etc. The idea is to maintain a consistent flow.
